### PR TITLE
dont install octokit via Gemfile

### PR DESCRIPTION
**Pull request by bastelfreak**
this PR also contains https://github.com/voxpupuli/modulesync/pull/226

### CODE REVIEW

**Review by ekohl — COMMENTED** [review]
Why is 4.9 in Gemfile while the gemspec has 4.0? It's all a bit inconsistent.

**Comment by bastelfreak** [comment]
I think there was a lot of copy and paste ongoing in the past. If you like I can add `'~> 4.9'` to the gemspec file.

File: Gemfile [hunk 1]
```diff
@@ -3,7 +3,6 @@ source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 gemspec
 
 gem 'cucumber'
```
Comment (ekohl) [inline Gemfile:5]: Just wondering: shouldn't this be in the gemspec as a dev dependency?

**Review by ekohl — APPROVED** [review]
I'm fine not updating it in the gemspec. It's not bound to it and it's probably OK. Technically we could verify what really is the minimum version, but I don't see anyone doing so.

**Pull request merged**